docs/_summary.md

```diff
@@ -118,6 +118,7 @@
     * [Overview](breaking_changes.md)
     * [My Pull Request Was Flagged](breaking_changes_instructions.md)
     * History
+      * [2020 Aug 30](ChangeLog/20200829.md)
```

Comment:
```suggestion
      * [2020 Aug 29](ChangeLog/20200829.md)
```
?

Reply:
I don't know how. I double checked it, I swear.